src/lib/utils/deepLinking.native.js

```diff
@@ -31,12 +31,10 @@ class DeepLinkingNative {
   pathname = ''
 
   subscribe = navigationCallback => {
+    this._unsubscribe = branch.subscribe(this._listener)
```

Comment:
why if check was removed ? you've fixed subscription logic in the commits before, why you've returned back ?

Reply:
It's related to the other comment.

Reply:
could you provide more details please (e.g. link to the comment you mean) ?

Reply:
https://github.com/GoodDollar/GoodDAPP/pull/1413/files#r423693587

Reply:
@MathiasMoralesDixtra its not a link to the comment

Reply:
I've adjusted logic as it should be